Sources/WhisperKit/Core/AudioProcessor.swift

```diff
@@ -235,39 +235,43 @@ public class AudioProcessor: NSObject, AudioProcessing {
         }
 
         let frameCount = AVAudioFrameCount(audioFile.length)
-        guard let inputBuffer = AVAudioPCMBuffer(pcmFormat: audioFile.processingFormat, frameCapacity: frameCount),
+        let chunkSize: AVAudioFrameCount = 10_000_000
+        guard let inputBuffer = AVAudioPCMBuffer(pcmFormat: audioFile.processingFormat, frameCapacity: min(frameCount, chunkSize)),
```

Comment:
Nit, we use "chunk" often relating to the vad chunking, this will hopefully make it a little more distinct.
```suggestion

        // Read audio in 10mb increments to reduce the memory spike for large audio files
        let maxReadFrameSize: AVAudioFrameCount = 10_000_000
        guard let inputBuffer = AVAudioPCMBuffer(pcmFormat: audioFile.processingFormat, frameCapacity: min(frameCount, maxReadFrameSize)),
```